src/DivConformingFESpaces.jl

```diff
@@ -0,0 +1,77 @@
+function FESpaces.FESpace(model::DistributedDiscreteModel,
+                          reffe::Tuple{RaviartThomas,Any,Any};kwargs...)
+
+  cell_reffes = map_parts(model.models) do m
+    basis,reffe_args,reffe_kwargs = reffe
+    cell_reffe = ReferenceFE(m,basis,reffe_args...;reffe_kwargs...)
+  end
+  sign_flips=_generate_sign_flips(model,cell_reffes)
+  # map_parts(sign_flips) do sf
+  #   println(sf)
+  # end
+  spaces = map_parts(model.models,sign_flips) do m,sign_flip
+     FESpace(m,reffe,sign_flip;kwargs...)
```

Comment:
According to my review comments in https://github.com/gridap/Gridap.jl/pull/728, you will need to first create an instance of `CellFE` and then call `FESpace` with it.

Reply:
Done